.gitlab-ci.yml

```diff
@@ -841,6 +846,7 @@ test_backend_integration:
       - report_backend_integration_*.html
       - sysstat.log
       - sysstat.svg
+      - acceptance.*.gz
```

Comment:
Now it should work.

But I suggest 1) remove the `mv ${CI_PROJECT_DIR}/backend_logs/* .` line and 2) here collect `backend_logs/`. This way the artifact will be better stored in a nice folder.